core/metacling/src/TCling.cxx

```diff
@@ -626,6 +630,22 @@ extern "C" int TCling__AutoLoadCallback(const char* className)
    return ((TCling*)gCling)->AutoLoad(className);
 }
 
+extern "C" int TCling__AutoLoadLibraryForModules(const char* Name)
+{
+   // FIXME: We're excluding some libraries from autoloading. Adding annotations to
+   // pcms from LinkDef files would fix this workaround.
+   // dll: For windows
+   static constexpr std::array<const char*, 8> excludelibs = { {"libRooStats.so",
```

Comment:
Perhaps we can just store `libRooFitCore` (eg no extension) and strip the extension from `Name`.